### PR TITLE
Allow infra and pbench node disks to be configured for larger size

### PR DESCRIPTION
**Pull request by akrzos**
For "smoke" testing the framework purposes a 32G disks should be fine,
however for the real deal large scale test we should use 100G for infra
and 500G for pbench.

### CODE REVIEW

File: roles/rhcos-post-install/templates/infra-node-machineset.yml.j2 [hunk 1]
```diff
@@ -36,13 +36,18 @@ items:
               filters: null
               id: {{rhcos_ami_id.stdout}}
             apiVersion: awsproviderconfig.k8s.io/v1alpha1
+            blockDevices:
+            - ebs:
+                iops: 0
```
Comment (chaitanyaenr) [inline roles/rhcos-post-install/templates/infra-node-machineset.yml.j2]: Think we should parameterize the iops field as well.

Comment (akrzos) [reply roles/rhcos-post-install/templates/infra-node-machineset.yml.j2]: Got it, I'll add them as well

**Comment by akrzos** [comment]
@chaitanyaenr updated

**Review by chaitanyaenr — APPROVED** [review]
/lgtm

**Pull request merged**